src/cipher/encoder/encoder.go

```diff
@@ -32,7 +32,10 @@ Todo:
 
 // TODO: constant length byte arrays must not be prefixed
 
-// EncodeInt encodes int
+// EncodeInt encodes an Integer type contained in `data`
+// into buffer `b`. If `data` is not an Integer type,
+// error message `log.Panic("PushAtomic, case not handled")`
```

Comment:
I dont think we should put the error message in the comments like that.
Its just unnecessary extra maintenance work if the message were to be changed.
I would trust the code more than a comment to see what error is returned. 
The comment would be just to get a better idea of what the function is doing.

Can just say that it panics instead of putting `log.Panic("PushAtomic, case not handled")` in the comments